mava/utils/make_env.py

```diff
@@ -65,6 +75,12 @@
 _jaxmarl_registry: Dict[str, Type[JaxMarlWrapper]] = {"Smax": SmaxWrapper, "MaBrax": MabraxWrapper}
 _gigastep_registry = {"Gigastep": GigastepWrapper}
 
+_gym_registry = {
+    "RobotWarehouse": GymWrapper,
+    "LevelBasedForaging": GymWrapper,
+    "Starcraft": SmacWrapper,
```

Comment:
For when we update the config. 

```suggestion
    "SMAC": SmacWrapper,
```